docs/api/console.mdx

```diff
@@ -686,3 +686,695 @@ Device removed from label successfully
+```
+
+<Tabs
+  block={true}
+  defaultValue="request"
+  values={[{"label":"Request","value":"request"},{"label":"Response","value":"response"}]}>
+<TabItem value="request">
+
+_Headers_
+
+| Parameter Name     | Type     | Description                          |
+| ------------------ | -------- | ------------------------------------ |
+| key (required)    | _string_ | your API key |
+| content-type (required) | _string_ | application/json              |
+
+_Body Parameters_
+
+| Parameter Name     | Type     | Description                          |
+| ------------------ | -------- | ------------------------------------ |
+| integration_id (required)    | _string_ | UUID of integration|
+| label_id / device_id (required) | _string_ | UUID           |
+| Function_id (optional) | _string_ | UUID             |
+
+</TabItem>
+<TabItem value="response">
+
+201: Created
+
+```
+{
+    "devices": [],
+    "id": "8bcf235a-0ee7-4e74-9856-e07c13a863b6",
+    "labels": [],
+    "name": "name"
+}
+
+```
+
+</TabItem>
+</Tabs>
+
+---
+
+### Delete flow
+
+```
+DELETE https://staging-console.helium.wtf/api/v1/flows/:flow_id
+```
+
+<Tabs
+  block={true}
+  defaultValue="request"
+  values={[{"label":"Request","value":"request"},{"label":"Response","value":"response"}]}>
+<TabItem value="request">
+
+_Headers_
+
+| Parameter Name     | Type     | Description                          |
+| ------------------ | -------- | ------------------------------------ |
+| key (required)    | _string_ | your API key |
+
+_Path Parameters_
+
+| Parameter Name | Type     | Description                      |
+| -------------- | -------- | -------------------------------- |
+| flow_id (required)  | _string_ | UUID of flow |
+
+</TabItem>
+<TabItem value="response">
+
+200: OK
+
+```
+Flow deleted successfully
+```
+
+</TabItem>
+</Tabs>
+
+---
+
+### Create Device (update existing)
+
+```
+POST https://staging-console.helium.wtf/api/v1/devices
+```
+
+<Tabs
+  block={true}
+  defaultValue="request"
+  values={[{"label":"Request","value":"request"},{"label":"Response","value":"response"}]}>
+<TabItem value="request">
+
+_Headers_
+
+| Parameter Name     | Type     | Description                          |
+| ------------------ | -------- | ------------------------------------ |
+| key (required)    | _string_ | your API key |
+| content-type (required) | _string_ | application/json              |
+
+_Body Parameters_
+
+| Parameter Name     | Type     | Description                          |
+| ------------------ | -------- | ------------------------------------ |
+| name (required)    | _string_ | A human-friendly name for the device |
+| app_eui (required) | _string_ | LoRaWAN Application EUI              |
+| app_key (required) | _string_ | LoRaWAN Application Key              |
+| dev_eui (required) | _string_ | LoRaWAN Device EUI                   |
+| Config_profile_id (required) | _string_ | Config profile uuid to attach                   |
```

Comment:
Does this parameter actually start with a capital C? This would diverge from the norm.

Reply:
updated case